_templates/page-templates/overview.md

```diff
@@ -7,7 +7,6 @@ title: Templates
 category: Templates
 lead: These templates represent common component combinations and are a great starting point for prototyping or trying USWDS to see if it’s a good fit for your project. Every template in this section is built using USWDS default theme settings and utilities.
 type: docs
-in_page_nav: false
```

Comment:
It looks like the in-page nav may have already been removed from the templates overview page, but I would note that I don't think this page is long enough to warrant in-page nav.

Reply:
Removed in [cd074b6](https://github.com/uswds/uswds-site/pull/2595/commits/cd074b67d0a0f0cc64d8ac42066ca8b06c50793e)